app/services/set_participant_categories.rb

```diff
@@ -27,41 +27,49 @@ def set_participant_categories
   end
 
   def fip_participant_categories_feature_flag_active
-    ParticipantCategories.new(eligible_participants, fip_flag_active_ineligible_participants, contacted_for_info_participants, details_being_checked_participants)
+    ParticipantCategories.new(eligible_participants, fip_flag_active_ineligible_participants, fip_flag_active_withdrawn_participants, contacted_for_info_participants, details_being_checked_participants)
   end
 
   def fip_participant_categories_feature_flag_inactive
-    ParticipantCategories.new([], [], contacted_for_info_participants, fip_flag_inactive_details_being_checked_participants)
+    ParticipantCategories.new([], [], withdrawn_participants, contacted_for_info_participants, fip_flag_inactive_details_being_checked_participants)
   end
 
   def cip_participant_categories
-    ParticipantCategories.new(cip_eligible_participants, [], contacted_for_info_participants, [])
+    ParticipantCategories.new(cip_eligible_participants, [], withdrawn_participants, contacted_for_info_participants, [])
   end
 
   def active_participant_profiles
     ParticipantProfilePolicy::Scope.new(user, school_cohort.active_ecf_participant_profiles).resolve
   end
 
   def ineligible_participants
-    active_participant_profiles.ineligible_status.includes(:user).order("users.full_name")
+    active_participant_profiles.ineligible_status.includes(:user).order("users.full_name").where.not(training_status: "withdrawn")
```

Comment:
I don't see anything particularly wrong in your use of `not`, but probably I'd personally use a scope mostly to have a clearer name ('acknowledged'? 'admitted'?), but it's a nitpick + personal taste, nothing else.

Reply:
I'm more puzzled by your use of "withdrawn" as a string instead of using a symbol.

Reply:
Yeah, i should have used a symbol. I'll change it.